tests/workflow/smoke.tests.txt

```diff
@@ -0,0 +1,75 @@
+# Number of tests
```

Comment:
Seems file naming is inconsistent here. Other workflows have file name `eden.<something>.tests.txt` but smoke tests are without `eden.` prefix - maybe we could drop these eden file name prefixes?